### ISSUE TITLE
about my own model

### ISSUE THREAD

**Issue by Sunkey2333** [kennymckormick/pyskl#168]
Thank you very much for your work. I have gone through all the issues and readme, as well as searched online. I noticed that everyone is using the pre-trained algorithms you provided to train their own datasets. I want to use the 2D or 3D skeleton provided in pyskl to train my own algorithm/model. How can I do this? Where should I make the necessary changes?Does your statement of supported algorithms mean that only these models can be trained using pyskl? And if my model is based on one of the supported algorithms, can I train it using pyskl? Also, if my server only has 4 GPUs, which file should I modify to adjust the batch size and learning rate? Thank you for your answer.

**Comment by Sunkey2333** [comment]
Can I understand from the contributing section that adding support for new algorithms to this toolbox is also considered a significant contribution, indicating that this toolbox may not be easily adaptable to new algorithms?

**Comment by kennymckormick** [comment]
Hi, @Sunkey2333 ,
You can train / test existing algorithm supported by PYSKL with this framework, and you can also support your own with the framework (will be quite easy) and you are welcome to PR new algorithms in PYSKL. I do not consider adding a new skeleton action recognition algorithm as a significant change, so you can do it your own. Significant change means supporting a new type of task (like future skeleton prediction, skeleton action detection, etc.)
When you use a different number of GPU for training, you should use linear scaling rule: make `LR / (num_gpu * batch_size_per_gpu)` as a constant value.
